wagtail_streamfield_migration_toolkit/test/tests/test_migrations.py

```diff
@@ -0,0 +1,353 @@
+import json
+import datetime
+from django.test import TestCase
+from django.db import connection
+from django.db.migrations.loader import MigrationLoader
+from django.db.migrations import Migration
+from django.db.models import JSONField, F
+from django.db.models.functions import Cast
+
+from .. import factories, models
+from wagtail_streamfield_migration_toolkit.operations import (
+    RenameStreamChildrenOperation,
+)
+from wagtail_streamfield_migration_toolkit.migrate_operation import MigrateStreamData
+
+
+class BaseMigrationTest(TestCase):
+    model_name = "SampleModel"
+
+    def apply_migration(self, revisions_from=None):
+        migration = Migration(
+            "test_migration", "wagtail_streamfield_migration_toolkit_test"
+        )
+        migration_operation = MigrateStreamData(
+            "wagtail_streamfield_migration_toolkit_test",
+            self.model_name,
+            "content",
+            [
+                (
+                    RenameStreamChildrenOperation(
+                        old_name="char1", new_name="renamed1"
+                    ),
+                    "",
+                )
+            ],
+            revisions_from=revisions_from,
+        )
+        migration.operations = [migration_operation]
+
+        loader = MigrationLoader(connection=connection)
+        loader.build_graph()
+        project_state = loader.project_state()
+        schema_editor = connection.schema_editor(atomic=migration.atomic)
+        migration.apply(project_state, schema_editor)
+
+
+class TestNonPageModelWithoutRevisions(BaseMigrationTest):
+    model_name = "SampleModel"
+
+    @classmethod
+    def setUpTestData(cls):
+        instances = [None for i in range(3)]
```

Comment:
Why not build up the list as you create each instance? Creating a list with `None`s first feels slightly confusing.

Additionally, what is this/`cls.instances` used for? It doesn't look like it's used any more.